e2e/commands/add-many.e2e.2.js

```diff
@@ -39,7 +39,58 @@ describe('bit add many programmatically', function () {
       expect(nodeStartOutputObj[0].addedComponents[0].id).to.equal('add_many_test_files/c');
     });
   });
-  describe('should add many components programatically', function () {
+  describe('should add many components programatically, process.cwd() is inside project path', function () {
+    before(function () {
+      helper.reInitLocalScope();
+      helper.copyFixtureComponents('add-many');
+      helper.npmLink('bit-bin');
+      helper.npmLink('bit-bin', path.join(helper.localScopePath, 'add_many_test_files'));
+      const innerScriptPathRelative = 'add_many_test_files/inner_folder';
+      const innerScriptPathAbsolute = path.join(helper.localScopePath, innerScriptPathRelative);
+
+      const scriptAbsolutePath = path.join(innerScriptPathAbsolute, 'add_components_programmatically.js');
+      helper.npmLink('bit-bin', innerScriptPathAbsolute);
+      nodeStartOutput = helper.nodeStart(`${scriptAbsolutePath} ${helper.localScopePath}`, process.cwd());
```

Comment:
I think the second arg should be the dir of the scriptAbsolutePath (aka innerScriptPathAbsolute) since what you are testing is that relative paths from the script are working